examples/hooks/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useBreadcrumb, UseBreadcrumbProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type BreadcrumbProps = React.ComponentProps<'div'> & UseBreadcrumbProps;
+
+export function Breadcrumb(props: BreadcrumbProps) {
+  const { items, refine, canRefine, createURL } = useBreadcrumb(props);
+
+  return (
+    <div
+      className={cx(
+        'ais-Breadcrumb',
+        !canRefine && 'ais-Breadcrumb--noRefinement',
+        props.className
+      )}
+    >
+      <ul className="ais-Breadcrumb-list">
+        <li
+          className={cx(
+            'ais-Breadcrumb-item',
+            !canRefine && 'ais-Breadcrumb-item--selected'
+          )}
+        >
+          <a
+            href={createURL(null)}
+            onClick={(event) => {
+              event.preventDefault();
+              refine(null);
```

Comment:
We should wrap this in `if (!isModifierClick(event))` to allow <kbd>Cmd</kbd> + Click (accessibility). (Same applies below.)

Reply:
Good point, thanks!